common/views/components/ViewerStructuresPrototype/ViewerStructuresPrototype.tsx

```diff
@@ -24,22 +26,52 @@ const ViewerStructuresPrototype: FunctionComponent<Props> = ({
   const canvases = manifest ? getCanvases(manifest) : [];
   const groupedStructures = groupStructures(canvases, structures);
 
+  const List = styled.ul.attrs({
+    className: classNames({
+      'plain-list no-margin no-padding': true,
+    }),
+  })`
+    border-left: 1px solid ${props => props.theme.color('pewter')};
+  `;
+
+  const Item = styled(Space).attrs({
+    as: 'li',
+    v: { size: 'xs', properties: ['padding-top', 'padding-bottom'] },
+    h: { size: 'm', properties: ['padding-left', 'padding-right'] },
+    className: classNames({
+      [font('hnl', 5)]: true,
+    }),
+  })<{ isActive: boolean }>`
+    position: relative;
+    cursor: pointer;
+
+    ${props =>
+      props.isActive &&
+      `
+        background: #222; // FIXME: we don't have a shade between dark-charcoal and black in the palette (light-black?)
+
+        &:before {
+          content: '';
+          position: absolute;
+          top: 0;
+          left: -1px;
+          bottom: 0;
+          width: 4px;
+          background: ${props.theme.color('yellow')};
+        }
+      `}
+  `;
+
   return groupedStructures.length > 0 ? (
-    <ul>
+    <List>
       {groupedStructures.map((structure, i) => {
         const firstCanvasInRange = structure.canvases[0];
         const canvasIndex = canvases.findIndex(
           canvas => canvas['@id'] === firstCanvasInRange
         );
         return (
-          <li key={i}>
+          <Item key={i} isActive={activeIndex === canvasIndex}>
             <a
```

Comment:
Just thinking that these should probably be buttons not links and would make them keyboard accessible - which I don't think the are currently

Reply:
yeah, I was thinking along the same lines. Will update